app/models/katello/content_view_version.rb

```diff
@@ -109,6 +109,14 @@ def description
       history.publish.successful.first.try(:notes)
     end
 
+    def description=(new_description)
+      if description
```

Comment:
Just to be clear, you're suggesting creating a successful publish in the history if someone tries to set the description on the content view version? Wouldn't that create bad data (i.e. the version would show as successfully published when it wasn't yet)?

I'd suggest an alternative of throwing some exception, perhaps, or some other non-database altering action than to include an added side-effect that some later developer, when using this method, wouldn't expect from a setter method.